### ISSUE TITLE
SOCKET.IO INTEGRATION

### ISSUE THREAD

**Issue by iamshour** [chatscope/use-chat#12]
Hello! Your work is awesome, but could you please provide an example of how an integration between this hook with socket.io server?

**Comment by jackytank** [comment]
indeed, It would be great if use-chat library can provide example of integrating simple WebSocket integration just like this one: https://github.com/JayaramachandranAugustin/ChatApplication

I tried hard to implement the public chat room that everyone can chat based on the use-chat example, anyone has any suggestion onto implement this feature?
